### PR TITLE
Add render cell render method and test

### PR DESCRIPTION
**Pull request by coreycartercodes**

### CODE REVIEW

File: lib/cell.rb [hunk 1]
```diff
@@ -21,7 +21,46 @@ def fired_upon?
   end
 
   def fire_upon
-    @fired_upon = true
+    case ship
+    when nil
+              #May need to add "Miss" here
```
Comment (coreycartercodes) [inline lib/cell.rb:26]: Thinking ahead for 2 potentials:
1 - "You Missed!"
2 - "You've already fired at that spot"

File: test/cell_test.rb [hunk 2]
```diff
@@ -37,4 +39,37 @@ def test_fired_upon?
...
 
+  def test_health_fire_upon
+    @cell1.place_ship(@cruiser)
+    assert_equal 3, @cell1.ship.health
+
+    @cell1.fire_upon
+    assert_equal 2, @cell1.ship.health
+  end
+
+  def test_render         #Should we break this up?
```
Comment (coreycartercodes) [inline test/cell_test.rb:50]: Unsure of best practice, should we break this up into multiple tests as it is all within the same method?

Comment (carson-jardine) [reply test/cell_test.rb:50]: I think we can keep it in one since it is testing the same the same method!

File: lib/cell.rb (hunk 1 above)
Comment (carson-jardine) [reply lib/cell.rb:26]: Good thinking, made a note to add these

**Pull request merged**